source/dem/find_boundary_cells_information.cc

```diff
@@ -72,6 +72,11 @@ BoundaryCellsInformation<dim>::build(
         boundary_cells_information,
         global_boundary_cells_information);
     }
+  else
+    pcout << "Warning: expansion of particle-wall contact list is disabled. "
```

Comment:
Those messages are very nice, but can you make it so that they only appear once and not at every load balance stage? you can have a boolean like "warning_displayed" as a member of the class.